recipe/meta.yaml

```diff
@@ -1,55 +1,59 @@
-{% set name = "HoloPy" %}
-{% set version = "3.2.1" %}
-{% set sha256 = "6994fb24ef20dabdb016ee0e624e474507f5abd48c6135837b09ccccc5b6ee77" %}
+{% set name = "holopy" %}
+{% set repo = "ralex0" %}
+{% set branch = "master" %}
+{% set sha256 = "b4fe3ca671b1ec5dfec8da9093ba5a1f3b60198de001b0fba0b3820d89a5e579" %}
 
 package:
   name: {{ name|lower }}
-  version: {{ version }}
+  version: {{ branch }}
 
 source:
-  fn: {{ name|lower }}-{{ version }}.tar.gz
-  url: https://github.com/manoharan-lab/{{ name }}/archive/{{ version }}.tar.gz
+  fn: {{ name|lower }}-{{ branch }}.tar.gz
+  url: https://github.com/{{ repo }}/{{ name }}/archive/{{ branch }}.tar.gz
   sha256: {{ sha256 }}
 
 build:
   number: 0
-  skip: True  # [py2k or win]
-  script: python setup.py install --single-version-externally-managed --record record.txt
+  skip: True  # [py2k]
+  script: "{{ PYTHON }} setup.py install"
 
 requirements:
   build:
+    - {{ compiler('c') }}
+    - {{ compiler('fortran') }}  # [not win]
+  host:
     - python
-    - numpy 1.12.*
+    - pip
+    - {{ pin_compatible('numpy') }}
     - setuptools
-    - gcc
-    - toolchain3
-
+    - m2w64-toolchain  # [win]
+    - nose
```

Comment:
I believe this should be a run dependency, not a host dependency.

Reply:
changed in @1b122e9